docs/drash/v2.x/2_tutorials/6_services/5_recommended_services.md

```diff
@@ -0,0 +1,46 @@
+# Recommended Services
+
+If you plan on creating a Drash application, there are a few services we have created, that we believe are part of a 'recommended set' to use in your application. Why? Because these services have many benefits for every single Drash application. 
+
+If you aren't familiar with services, please read the [services documentation](https://drash.land/drash/v2.x/tutorials/services/introduction) before reading on.
+
+## List of Services to Use
+
+These services can be very useful when added by default because they can add builtin logging, or ways to secure your server or system. Therefore we believe that every Drash application should use the below services, *but not required!*. The choice is still yours!
+
+1. [`paladin`](https://drash.land/drash/v2.x/tutorials/services/drash-approved-services/paladin)
+2. [`dexter`](no docs :sadface:)
```

Comment:
do we want to wait for this?

Reply:
yeah will wait for dexter to bne added :)